apps/docs/build.md

```diff
@@ -217,7 +217,7 @@ the following:
 
 0. Make sure you have `build_apps: true` and `use_my_apps: true` in locals.yml.
 
-1. set `pretty_js: false` in locals.yml. This will make dashboard and pegasus
+1. set `optimize_webpack_assets: true` in locals.yml. This will make dashboard and pegasus
```

Comment:
For later: Maybe we should throw this into locals.yml.default with documentation there.